docs/api.rst

```diff
@@ -13689,4 +13689,152 @@ Refetch EVM transactions for a specific time period
+
+        {
+          "message": "",
+          "result": {
+            "chainId": 1,
+            "from": "0xc37b40ABdB939635068d3c5f13E7faF686F03B65",
+            "to": "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84",
+            "data": "0xa9059cbb0000000000000000000000009531c059098e3d194ff87febb587ab07b30b13060000000000000000000000000000000000000000000000000000000000000001"
+          },
+          "status_code": 200
+        }
+
+    :resjson object result: The transaction data needed for the token transfer
+    :resjson integer chainId: The chain ID of the blockchain network
+    :resjson string from: The sender's address
+    :resjson string to: The token contract address
+    :resjson string data: The encoded transaction data for the token transfer
+    :statuscode 200: Transaction data successfully prepared
+    :statuscode 400: Malformed request
+    :statuscode 409: Error preparing the payload
+    :statuscode 500: Internal Rotki error
+
+  .. http:post:: /api/(version)/wallet/transfer/native
+
+    Prepares a native cryptocurrency transfer transaction without submitting it to the blockchain. This endpoint returns the transaction data needed to perform a transfer of the blockchain's native asset (e.g., ETH).
+
+    **Example Request:**
+
+      .. http:example:: curl wget httpie python-requests
+
+        POST /api/1/wallet/transfer/native HTTP/1.1
+        Host: localhost:5042
+        Content-Type: application/json;charset=UTF-8
+
+        {
+          "from_address": "0xc37b40ABdB939635068d3c5f13E7faF686F03B65",
+          "to_address": "0x9531C059098e3d194fF87FebB587aB07B30B1306",
+          "amount": "0.0003",
+          "chain_id": "ethereum"
+        }
+
+    :reqjson string from_address: The address from which the native cryptocurrency will be sent
+    :reqjson string to_address: The address to which the native cryptocurrency will be sent
+    :reqjson string amount: The amount of native cryptocurrency to transfer as a string
+    :reqjson string chain_id: The chain id name (e.g., "ethereum" for Ethereum)
+
+    **Example Response:**
+
+      .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        {
+          "message": "",
+          "result": {
+            "from": "0xc37b40ABdB939635068d3c5f13E7faF686F03B65",
+            "to": "0x9531C059098e3d194fF87FebB587aB07B30B1306",
+            "value": 300000000000000,
+            "maxFeePerGas": 2000000000,
+            "maxPriorityFeePerGas": 1000000000,
+            "nonce": 55
+          },
+          "status_code": 200
+        }
+
+    :resjson object result: The transaction data needed for the native transfer
+    :resjson string from: The sender's address
+    :resjson string to: The recipient's address
+    :resjson integer value: The amount to transfer in the smallest unit of the native cryptocurrency
+    :resjson integer maxFeePerGas: The maximum fee per gas unit the user is willing to pay
```

Comment:
if you removed gas you need to remove it from here too. I am fine with having it remove for now, but "gave problems is not a good reason". We should be able to handle problems.

On the other hand I don't think gas is needed as it's something the user sets at the wallet anyway so am fine with the change. Having a way to set it already from rotki (the dapp) is not really common, though some dapps do. But in the end the wallet has the final say.

Reply:
was in the frontend side when sending it to the wallet. But as you said the wallet chooses it so no need to configure it. The error seems to come from how it is calculated in some chains